pnet_macros/src/decorator.rs

```diff
@@ -1154,6 +1154,8 @@ fn parse_ty(ty: &str) -> Option<(usize, Endianness, EndiannessSpecified)> {
             let e = e.as_str();
             if e == "be" {
                 (Endianness::Big, EndiannessSpecified::Yes)
+            } else if e == "he" {
+                (HOST_ENDIANNESS, EndiannessSpecified::Yes)
```

Comment:
I think we probably want a member of `Endianness` which is `Host`, rather than using little/big.

It's a bit more work, but means:

 * The generated doc comments say the right thing: https://github.com/libpnet/libpnet/blob/182604958009b86d85e80d687a5cca91db08c94b/pnet_macros/src/decorator.rs#L1208-L1213
 * No bitshifting is generated: https://github.com/libpnet/libpnet/blob/182604958009b86d85e80d687a5cca91db08c94b/pnet_macros/src/decorator.rs#L465-L482

If you ctrl-f for `Endianness` you should find the places that need changing.